etc/openqa/openqa.ini

```diff
@@ -85,7 +85,27 @@
 
 ## Authentication method to use for user management
 [auth]
-# method = Fake|OpenID
+# method = Fake|OpenID|OAuth2
+
+# for GitHub/salsa.debian.org and providers listed on https://metacpan.org/pod/Mojolicious::Plugin::OAuth2#Configuration
+# one can use:
+#[oauth2]
+#provider = github/debian_salsa
```

Comment:
I find it a bit hard to know what "github/debian_salsa" means, or in particular the meaning of the "/". To separate alternatives we should use "|" as above. And not sure what "debian_salsa" in particular means. Do we need to give a full URL? Or is this just an internal identifier?

Reply:
Using a `|` makes sense. `debian_salsa` is the provider one needs to specify to use preset-parameters for salsa.debian.org but one does not need to specify that domain name or a full URL. The decision to use short, lower-case provider names like `github` comes from the underlying Perl module `Mojolicious::Plugin::OAuth2` (see https://metacpan.org/pod/Mojolicious::Plugin::OAuth2#Configuration) so it make sense for use to stick with it.

Reply:
Agreed, i prefer `|` as well.

Reply:
Changed it to use `|`.